### DISCUSSION TITLE
Integrate iron-sesson v8

### DISCUSSION THREAD

**Discussion by TimKieu** [cloudflare/next-on-pages#705] [Q&A]
@all Dear All. I am building next-on-pages webapps with Next-Auth V5 and Iron-session v8. Both of the libs are edge-runtime ready, but I always got error 405 when deploy and run on CF Pages.
Does any one have any suggestions for integration with Iron-session? Or alternative secure Auth options.
Thanks much.

**Comment by dario-piotrowicz** [comment]
If you're getting 405s then it can be related to this: https://github.com/cloudflare/next-on-pages/issues/679 😕
